test/tests/interfaces/pbs_sched_interface_test.py

```diff
@@ -41,22 +41,32 @@
 
 @tags('multisched')
 class TestSchedulerInterface(TestInterfaces):
+
     """
     Test suite to test different scheduler interfaces
     """
 
     def setUp(self):
         # delete all schedulers except default
-        self.server.manager(MGR_CMD_DELETE, SCHED, id="@default")
+        self.server.manager(MGR_CMD_DELETE,
+                            SCHED,
+                            id="@default")
+        self.server.manager(MGR_CMD_CREATE,
+                            SCHED,
+                            id="TestCommonSched")
         TestInterfaces.setUp(self)
 
     def test_create_successful_scheduler_object(self):
         """
         Creates a scheduler object succcessfully .
         """
-        a = {'sched_cycle_length': 12000}
-        rc = self.server.manager(MGR_CMD_CREATE, SCHED, a, id="test1")
-        self.assertTrue(rc == 0)
+        try:
+            self.server.manager(MGR_CMD_CREATE,
+                                SCHED,
+                                {'sched_cycle_length': 12000},
+                                id="testCreateSched")
+        except PbsManagerError, e:
+            self.fail(e.msg)
```

Comment:
Changed test cases as u said. 

Still see value is differentiating error and fail test cases.

BTW, tearDown will be called when test case has error? According to my knowledge interpreter exits. Will it be problem subsequent testcases ? I don't know.